common/utils/src/main/resources/error/error-conditions.json

```
@@ -1093,6 +1093,12 @@
     ],
     "sqlState" : "22008"
   },
+  "DATE_TIME_FIELD_OUT_OF_BOUNDS" : {
+    "message" : [
+      "The value '<badValue>' you entered for <unit> is not valid. Please provide a value between <range>. To disable strict validation, you can turn off ANSI mode by setting <ansiConfig> to false."
```

Use `toSQLValue` to quote `badValue`, and wrap `false`:
```suggestion
      "The value <badValue> you entered for <unit> is not valid. Please provide a value in the range <range>. To disable strict validation, you can turn off ANSI mode by setting <ansiConfig> to \"false\"."
```